packages/react-app/src/components/Options/SelectMenu.tsx

```diff
@@ -48,13 +48,18 @@ const SelectMenu: React.FC<SelectMenuProps> = ({ format, changeOptions }) => (
       },
       Root: {
         style: () => ({
-          width: "60%",
+          width: "65.55%",
           height: "10px",
           marginTop: "10px",
           marginLeft: "1px",
           padding: "0px",
           background: colors.textFieldGrey,
           borderRadius: "2px",
+          ["@media screen and (max-width: 414px)"]: { width: "54.2%" },
```

Comment:
(Ideally this would be continuous, but after looking into it, seems there isn't an easy solution and is okay for now).